 env_type: 'fake'
 max_epoch_steps: [10]
 learning_rate: [0.00001]
 gamma: [0.9]
 total_timesteps: 1000000
 model_types: ['td3']
